lib/libzutil/os/freebsd/zutil_import_os.c

```diff
@@ -249,6 +249,6 @@ zfs_dev_flush(int fd __unused)
 }
 
 void
-update_vdevs_config_dev_sysfs_path(nvlist_t *config)
+update_vdevs_config_dev_sysfs_path(nvlist_t *config __unused)
```

Comment:
Why `__unused` here and not `(void) config`?

Reply:
Because there's __unused above, too, but fair point, I've changed them both to void casts.